modules/ROOT/examples/live-demos/comments-callback-with-mentions/index.js

```diff
@@ -211,157 +122,153 @@ import ('https://cdn.jsdelivr.net/npm/@faker-js/faker@9/dist/index.min.js').then
-  
-  tinymce.init({
-    selector: 'textarea#comments-callback-with-mentions',
-    license_key: 'gpl',
-    toolbar: 'addcomment showcomments code | bold italic underline',
-    menubar: 'file edit view insert format tools tc help',
-    menu: {
-      tc: {
-        title: 'TinyComments',
-        items: 'addcomment showcomments deleteallconversations'
-      }
-    },
-    plugins: [ 'tinycomments', 'mentions', 'help', 'code', 'quickbars', 'link', 'lists', 'image' ],
-    quickbars_selection_toolbar: 'alignleft aligncenter alignright | addcomment showcomments',
-    quickbars_image_toolbar: 'alignleft aligncenter alignright | rotateleft rotateright | imageoptions',
-    tinycomments_mentions_enabled: true,
-    sidebar_show: 'showcomments',
-  
-    mentions_item_type: 'profile',
-    mentions_min_chars: 0,
-    mentions_selector: '.mymention',
-    mentions_fetch,
-    mentions_menu_hover,
-    mentions_menu_complete,
-    mentions_select,
-  
-    tinycomments_mode: 'callback',
-    tinycomments_author: currentUser.id,
-    tinycomments_author_name: currentUser.fullName,
-    tinycomments_avatar: currentUser.image,
-    tinycomments_create,
-    tinycomments_reply,
-    tinycomments_delete,
-    tinycomments_resolve,
-    tinycomments_delete_all,
-    tinycomments_lookup,
-    tinycomments_delete_comment,
-    tinycomments_edit_comment,
-    tinycomments_fetch,
-    tinycomments_fetch_author_info
+const tinycomments_fetch = (conversationUids, done) => {
+  const fetchedConversations = {};
+  conversationUids.forEach((uid) => {
+    const conversation = conversationDb[uid];
+    if (conversation) {
+      fetchedConversations[uid] = {...conversation};
+    }
   });
-});
+  setTimeout(() => done({ conversations: fetchedConversations }), fakeDelay);
+};
+
+tinymce.init({
+  selector: 'textarea#comments-callback-with-mentions',
+  license_key: 'gpl',
+  toolbar: 'addcomment showcomments code | bold italic underline',
+  menubar: 'file edit view insert format tools tc help',
+  menu: {
+    tc: {
+      title: 'TinyComments',
+      items: 'addcomment showcomments deleteallconversations'
+    }
+  },
+  plugins: [ 'tinycomments', 'mentions', 'help', 'code', 'quickbars', 'link', 'lists', 'image' ],
+  quickbars_selection_toolbar: 'alignleft aligncenter alignright | addcomment showcomments',
+  quickbars_image_toolbar: 'alignleft aligncenter alignright | rotateleft rotateright | imageoptions',
+  
+  tinycomments_mode: 'callback',
+  tinycomments_mentions_enabled: true,
+  sidebar_show: 'showcomments',
+  tinycomments_create,
+  tinycomments_reply,
+  tinycomments_delete,
+  tinycomments_resolve,
+  tinycomments_delete_all,
+  tinycomments_lookup,
+  tinycomments_delete_comment,
+  tinycomments_edit_comment,
+  tinycomments_fetch,
+
+  mentions_item_type: 'profile',
+  mentions_min_chars: 0,
+  mentions_selector: '.mymention',
+  mentions_fetch,
+  mentions_menu_hover,
+  mentions_menu_complete,
+  mentions_select,
+  user_id,
```

Comment:
```suggestion
  tinycomments_fetch_author_info: async (done) => await fetch(`${API_URL}/${id}`)
    .then((response) => response.json())
    .then((userInfo) => {
      done({
        author: userInfo.id,
        authorName: userInfo.name,
        authorAvatar: userInfo.avatar
      });
    })
```
This callback was made for this purpose in 7. Please add this to each of the comments demos